lib/keypair.ex

```diff
@@ -49,9 +50,11 @@ defmodule Elixium.KeyPair do
         private = Mnemonic.to_entropy(phrase)
         {pub, priv} = get_from_private(private)
         create_keyfile({pub, priv})
+        {pub, priv}
       else
         {pub, priv} = get_from_private(phrase)
         create_keyfile({pub, priv})
+        {pub, priv}
```

Comment:
Since both of these statements contain `create_keyfile/1` and both return `{pub, priv}` they should be moved outside of the if statement into the end of the function body.

Reply:
Easy